app/helpers/application_helper/button/physical_storage_timeline.rb

```diff
@@ -0,0 +1,14 @@
+class ApplicationHelper::Button::PhysicalStorageTimeline < ApplicationHelper::Button::EmsTimeline
+  def calculate_properties
+    super
+    self[:hidden] = true unless visible?
+  end
+
+  def visible?
+    @record.supports?(:timeline)
+  end
```

Comment:
@TonySchneider these two methods look identical to the ones in `ApplicationHelper::Button::EmsTimeline` so you don't have to override them